lib/src/main/kotlin/com/lemonappdev/konsist/api/verify/KoDeclarationAndProviderAssert.kt

```diff
@@ -147,6 +147,120 @@ fun <E : KoBaseProvider> Sequence<E?>.assertFalse(
     this.toList().assert(strict, additionalMessage, suppressName, function, false)
 }
 
+/**
+ * Asserts that the element has `null` value.
+ *
+ * @param additionalMessage An optional message to provide additional context when the assertion fails.
+ *                          This message will be included in the assertion error if the assertion fails.
+ * @param suppressName An optional suppress name. By default, suppress name is derived from JUnit method name,
+ *                     however for koTest framework it hase to be manually specified to allow suppression.
+ */
+fun <E : KoBaseProvider> E?.assertNull(
+    additionalMessage: String? = null,
+    suppressName: String? = null,
+): Unit {
+    listOf(this).assert(true, additionalMessage, suppressName, isEmptyOrNull = true, onSingleElement = true)
+}
+
+/**
+ * Asserts that the element has not `null` value.
+ *
+ * @param additionalMessage An optional message to provide additional context when the assertion fails.
+ *                          This message will be included in the assertion error if the assertion fails.
+ * @param suppressName An optional suppress name. By default, suppress name is derived from JUnit method name,
+ *                     however for koTest framework it hase to be manually specified to allow suppression.
+ */
+fun <E : KoBaseProvider> E?.assertNotNull(
+    additionalMessage: String? = null,
+    suppressName: String? = null,
+): Unit {
+    listOf(this).assert(true, additionalMessage, suppressName, isEmptyOrNull = false, onSingleElement = true)
+}
+
+/**
```

Comment:
Is x`strict`correctly described in Kock?

Reply:
If set to `true`, the assertion throws an exception list is empty or contains only null values.